packages/web-app-files/src/components/SideBar/Shares/FileShares.vue

```diff
@@ -28,7 +28,7 @@
         <li v-for="collaborator in displayCollaborators" :key="collaborator.key">
           <collaborator-list-item
             :share="collaborator"
-            :modifiable="!collaborator.indirect"
+            :modifiable="shareIsModifiable(collaborator)"
```

Comment:
```suggestion
            :modifiable="isShareModifiable(collaborator)"
```
I personally like when methods with boolean return type have `is`, `are`, etc as prefix of the method name, not in between.